game/src/game/components/Controllers/PlayerController.cpp

```diff
@@ -21,7 +21,8 @@ static constexpr size_t kGamepadId = GLFW_JOYSTICK_1;
 static constexpr float kRespawnSeconds = 3.0f;
 static constexpr float kDefaultBrake = 0.0f;
 static constexpr float kSpeedMultiplier = 0.1f;
-static constexpr float kHanldingMultiplier = 0.0f;
+static constexpr float kHandlingMultiplier = 0.0f;
+static float shoot_cooldown;
```

Comment:
Can this be made a member var of this class instead of static? If we do have time to throw in split screen, every instance of a player should have their own independent cooldown

Reply:
good point !